pkg/kafka/consumergroup/validators.go

```diff
@@ -0,0 +1,33 @@
+package consumergroup
+
+import (
+	"errors"
+	"regexp"
+
+	"github.com/redhat-developer/app-services-cli/pkg/common/commonerr"
+	"github.com/redhat-developer/app-services-cli/pkg/localize"
+)
+
+const (
```

Comment:
Should this be correct regexp? ( alphanumeric and hyphen).

Reply:
Asked in like that contains regexp